Sources/TensorFlow/Loss.swift

```diff
@@ -24,6 +24,23 @@ public func meanSquaredError<Scalar: TensorFlowFloatingPoint>(
     return (expected - predicted).squared().mean()
 }
 
+/// Computes the mean squared logarithmic error between predictions and expectations.
+///
+/// Note that negative Tensor entries will be clamped at 0 to avoid undefined
```

Comment:
```suggestion
/// - Note: Negative Tensor entries will be clamped at 0 to avoid undefined
```

We can use the markup syntax for notes in documentation comments. These will be properly rendered in API documentation.

Reply:
Wow I did not think of even looking that up. Thank you for teaching me that!

Reply:
Glad it helped!